apps/approval/views.py

```diff
@@ -116,9 +116,14 @@ def create_membership_application(request):
             )
             return redirect("profiles_active", active_tab="membership")
 
-        # Extend length of membership by 1 year
-        membership = AllowedUsername.objects.get(username=request.user.ntnu_username)
-        new_expiration_date = datetime.date(membership.expiration_date.year + 1, 9, 16)
+        # Grant membership until 16th of September this year if the request was sent previous to 1st of July,
+        # or until 16th of September next year if the request was sent after 1st of July
```

Comment:
Note: due to leap-years and such, we are not actually guaranteed for it to last until 16th of September the next year. I do not think this is something that we need to address

Reply:
Alright, so should it just specify that the applicant is granted membership for one year?

Reply:
No, I think this is fine!